platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/shared/CheckoutRequest.java

```diff
@@ -10,28 +10,30 @@
  *******************************************************************************/
 package org.eclipse.che.api.git.shared;
 
+import java.util.List;
+
 import org.eclipse.che.dto.shared.DTO;
 
 /**
- * Request to checkout a branch to the working tree.
+ * Request to checkout a branch / file / s to the working tree.
```

Comment:
Reformulate sentence without usage of '/'. 
It is hard to understand ' / s'. What do you mean? Files?